src/super_gradients/recipes/roboflow_yolo_nas_s.yaml

```diff
@@ -0,0 +1,98 @@
+# A recipe to fine-tune YoloNAS on Roboflow datasets.
+# Checkout the datasets at https://universe.roboflow.com/roboflow-100?ref=blog.roboflow.com
+#
+# `dataset_name` refers to the official name of the dataset.
+# You can find it in the url of the dataset: https://universe.roboflow.com/roboflow-100/digits-t2eg6 -> digits-t2eg6
+#
+# Example: python -m super_gradients.train_from_recipe --config-name=roboflow_yolo_nas_s dataset_name=digits-t2eg6
+
+defaults:
+  - training_hyperparams: coco2017_yolo_nas_train_params
+  - dataset_params: roboflow_detection_dataset_params
+  - checkpoint_params: default_checkpoint_params
+  - arch_params: yolo_nas_s_arch_params
```

Comment:
Same as my comment in src/super_gradients/recipes/coco2017_yolo_nas_s.yaml defaults.

Reply:
Updated in https://github.com/Deci-AI/super-gradients/commit/0d3f52b292883d11c25b1f81ccca427742688653